pkg/arvo/mar/hall/speeches.hoon

```diff
@@ -1,23 +0,0 @@
-::
```

Comment:
I'm a little skeptical about removing marks for system/standard-app types that are still available, just because they happen to not be in use. But this is mark is also marginal ... @Fang-?

Reply:
This is unused and I think it should be deleted.

Reply:
Pretty sure this is fine to go, yeah.